chia/util/make_test_constants.py

```diff
@@ -1,6 +1,10 @@
 from typing import Dict
 
-from chia.consensus.default_constants import DEFAULT_CONSTANTS, ConsensusConstants
+import chia.consensus.default_constants
+
+
+DEFAULT_CONSTANTS = chia.consensus.default_constants.DEFAULT_CONSTANTS
+ConsensusConstants = chia.consensus.default_constants.ConsensusConstants
```

Comment:
what rule prefers this way of importing names?

Reply:
Looks like this particular spot was me getting off on a wrong tangent...  sorry.  Undoing that bit, and tweaking a couple other.

The problem was at the other end of the import where we had `from .constants import ConsensusConstants` and then tried to import `ConsensusConstants` here.  The `no_implicit_reexport` is the issue there.  I'll switch the import to just get `ConsensusConstants` from the original place.

Thanks.